src/utils/humanReadableValue.ts

```diff
@@ -0,0 +1,5 @@
+import { BigNumber } from 'bignumber.js'
```

Comment:
there is quite similar (by purpose) function in `src/logic/tokens/utils`. Should we maybe move this one there too?